app/views/experiences/build/_sidebar.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="container-fluid">
+  <div class="d-flex justify-content-between pt-5">
+    <i class="fas fa-skull-crossbones text-primary"></i>
+    <a class="text-muted" href="/experiences"><small><u>Save and exit</u></small></a>
+  </div>
+  <h3 class="mt-5">Submit your<br>
+    experience</h3>
+  <div class="d-flex mt-5">
+    <ul class="nav nav-sm flex-column">
+      <li class="d-flex justify-content-between">
+        <a class="nav-link me-5" href="/experiences/<%= @experience.id %>/build/add_name">Time and duration</a>
+        <% if @experience.name && @experience.start_time && @experience.end_time %>
+          <i class="fas fa-check-circle my-auto"></i>
+        <% end %>
+      </li>
+      <li class="nav-item d-flex justify-content-between">
+        <a class="nav-link" href="/experiences/<%= @experience.id %>/build/add_description">What we'll do</a>
+        <% if @experience.description %>
+          <i class="fas fa-check-circle my-auto"></i>
+        <% end %>
+      </li>
+      <li class="nav-item d-flex justify-content-between">
+        <a class="nav-link" href="/experiences/<%= @experience.id %>/build/add_price">How much</a>
+        <% if @experience.price %>
+          <i class="fas fa-check-circle my-auto"></i>
+        <% end %>
+      </li>
+      <li class="nav-item d-flex justify-content-between">
+        <a class="nav-link" href="/experiences/<%= @experience.id %>/build/add_address">Where we'll do</a>
+        <% if @experience.address %>
+          <i class="fas fa-check-circle my-auto"></i>
+        <% end %>
+      </li>
+      <li class="nav-item d-flex justify-content-between">
+        <a class="nav-link" href="/experiences/<%= @experience.id %>/build/add_photos">Pictures</a>
```

Comment:
All of these `href`s should be using the rails_routes
<img width="941" alt="Screen Shot 2022-01-11 at 15 25 15" src="https://user-images.githubusercontent.com/25542223/148891989-c4ea5824-a7b4-42b1-ab86-306d8c5bbd2e.png">
instead of hard coded coding the path, let's use `link_to` instead